Makefile

```diff
@@ -781,10 +797,20 @@ COMMON_SRC = \
             io/vtx_string.c \
             io/vtx_smartaudio.c \
             io/vtx_tramp.c \
-            io/vtx.c \
+            io/vtx.c
+            
+COMMON_DEVICE_SRC = \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC)
 
+ifeq ($(OSD_SLAVE),yes)
+TARGET_FLAGS := -DUSE_OSD_SLAVE $(TARGET_FLAGS)
+COMMON_SRC := $(COMMON_SRC) $(OSD_SLAVE_SRC) $(COMMON_DEVICE_SRC)
+else
+TARGET_FLAGS := -DUSE_FC $(TARGET_FLAGS)
```

Comment:
I thought we'd got rid of all the `SKIP_`, but, checking, there are indeed a few remaining. That should be tidied up at some point.